src/theme.css

```diff
@@ -10,17 +12,35 @@
   --color-neutral-8: #2e2e2e;
   --color-neutral-9: #000000;
 
-  --color-primary: #f4f4f4;
-  --color-primary-text: #2e2e2e;
-  --color-primary-light: #f4f4f4;
-  --color-primary-light-text: #2e2e2e;
+  --color-error-1:#ffe2e2;
+  --color-error-2: #eb3737;
```

Comment:
neutral and error colors are also themeable. If you want a dark theme, you can invert the neutral colors. Then we can remove the duplicate `--color-error` vars